docs/pages/guides/smi-conformance.md

```diff
@@ -0,0 +1,163 @@
+
+Open a terminal and run the following command.
+
+ <pre class="codeblock-pre"><div class="codeblock">
+ <div class="clipboardjs">
+ mesheryctl mesh validate -a [name of the adapter] -t [path to token for authentication] -s smi
+ </div></div>
+ </pre>
+
+ For example to run an SMI Conformance test on Open Service Mesh, we can run:
+
+ <pre class="codeblock-pre"><div class="codeblock">
+ <div class="clipboardjs">
+ mesheryctl mesh validate -a meshery-osm:10009 -t ~/Downloads/auth.json -s smi
+ </div></div>
+ </pre>
+
+ To view the results of the test, you can open up Meshery in the browser and navigate to the "Conformance" tab.
+
+_SMI Conformance Test Results_
+
+<a href="{{ site.baseurl }}/assets/img/smi/smi-conformance-page.png"><img alt="SMI Conformance Test Results" src="{{ site.baseurl }}/assets/img/smi/smi-conformance-page.png" /></a>
+
+Click the dropdown button to view the results of a specific test.
+
+_Viewing the Results_
+
+<a href="{{ site.baseurl }}/assets/img/smi/smi-conformance-result.png"><img alt="SMI Conformance Test Results" src="{{ site.baseurl }}/assets/img/smi/smi-conformance-result.png" /></a>
+
+## Running SMI Conformance Tests in CI/CD Pipelines
+
+You can use [Meshery SMI Conformance GitHub action](https://github.com/layer5io/meshery-smi-conformance-action) to run SMI Conformance tests in your CI/CD pipelines.
+
+The conformance test would be run in your pipeline and the results will be published on your Meshery Dashboard (see [Setup Meshery and Install a Service Mesh](#setup-meshery-and-install-a-service-mesh)).
+
+Download the token from the Meshery Dashboard by clicking on the profile icon on the top-right corner.
+
+_Downloading the token_
+
+<a href="{{ site.baseurl }}/assets/img/smi/download-token.png"><img alt="SMI Conformance Test Results" src="{{ site.baseurl }}/assets/img/smi/download-token.png" /></a>
+
+You can use this token to authenticate the instance of Meshery running in your CI/CD workflow.
+
+{% include alert.html type="info" title="Using the token in GitHub workflows" content="You can use the <a href='https://docs.github.com/en/actions/reference/encrypted-secrets'>secrets feature in GitHub</a> to store the token." %}
+
+A sample workflow that could be added to your `.github/workflows` folder is given below. This example uses Open Service Mesh.
+
+ <pre class="codeblock-pre"><div class="codeblock">
+ <div class="clipboardjs">
+ name: SMI Conformance with Meshery
+ on:
+   push:
+     branches:
+       - 'main'
+ 
+ jobs:
+   smi-conformance:
+     name: SMI Conformance
+     runs-on: ubuntu-latest
+     steps:
+       - name: Setup MiniKube
+         uses: manusa/actions-setup-minikube@v2.4.1
+         with:
+           minikube version: 'v1.21.0'
+           kubernetes version: 'v1.20.7'
+           driver: docker
+ 
+       - name: Install OSM
+         run: |
+            curl -LO https://github.com/openservicemesh/osm/releases/download/v0.6.0/osm-v0.6.0-linux-amd64.tar. gz
```

Comment:
Ouch. @DelusionalOptimist are we ready to replace this with `mesheryctl mesh`?

Reply:
Waiting for the PR linked in the PR description. @leecalcote

Reply:
And once the PR is merged, we need to modify the action as well to use `mesheryctl mesh`. The issue currently being Meshery is not able to deploy the latest OSM.

Reply:
hehe. yeah just an inch away :pinching_hand: